src/Arcus.EventGrid.Tests.Integration/Arcus.EventGrid.Tests.Integration.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
```

Comment:
When specifying both net6 and net8 here, aren't the test then automatically run against both frameworks ?

Reply:
Yes, that's right, just wasn't sure to do this, as we don't do this for other repo's neither. For the unit tests, we can't do this just yet bc the Codecov requires a single framework, for the integration tests, it means a double test run time.

Reply:
So, we leave it as is for now.
However, I think it would be good to investigate this a bit further, so maybe create an issue for this ?

Reply:
Added it here: https://github.com/arcus-azure/arcus/issues/340